src/config/Tokens.js

```diff
@@ -432,3 +432,12 @@ export function getTokenBySymbol(chainId, symbol) {
 export function getWhitelistedTokens(chainId) {
   return TOKENS[chainId].filter((token) => token.symbol !== "USDG");
 }
+
+export function getVisibleTokens(chainId) {
+  return getWhitelistedTokens(chainId).filter((token) => {
+    if (token.isWrapped || token.isTempHidden) {
```

Comment:
probably matter of taste, maybe better `(token) => !token.isWrapper && !token.isTempHidden`? i think it's a bit easier to grasp the filter logic (at least for me)